engine/src/main/java/org/terasology/engine/world/block/internal/BlockManagerImpl.java

```diff
@@ -189,7 +191,7 @@ protected void registerFamily(BlockFamily family) {
 
     private void registerBlock(Block block, RegisteredState newState) {
         if (block.getId() != UNKNOWN_ID) {
-            logger.info("Registered Block {} with id {}", block, block.getId());
+            logger.atInfo().addArgument(() -> block).addArgument(() -> block.getId()).log("Registered Block {} with id {}");
```

Comment:
Simple getter method so impact neglectable.
```suggestion
            logger.info("Registered Block {} with id {}", block, block.getId()); //NOPMD
```